lib/puppet/provider/reboot/windows.rb

```diff
@@ -143,6 +144,19 @@ def package_installer_syswow64?
     end
   end
 
+  def pending_computer_rename?
+    path = 'SYSTEM\CurrentControlSet\Control\ComputerName'
```

Comment:
is this subject to registry redirection (as in due we need to check two places)?